easybuild/easyblocks/p/psi.py

```diff
@@ -136,8 +137,9 @@ def configure_step(self):
 
             ConfigureMake.configure_step(self, cmd_prefix=self.cfg['start_dir'])
         else:
-            self.cfg['configopts'] += "-DPYTHON_INTERPRETER=%s " % os.path.join(pythonroot, 'bin', 'python')
-            self.cfg['configopts'] += "-DCMAKE_BUILD_TYPE=Release "
+            self.log.info("Using CMake based build")
+            self.cfg['configopts'] += " -DPYTHON_INTERPRETER=%s " % os.path.join(pythonroot, 'bin', 'python')
+            self.cfg['configopts'] += "-DCMAKE_BUILD_TYPE=RelWithDebInfo "
```

Comment:
also: is just changing the build type OK? should we make that a separate parameter?